docs/glossary.md

```diff
@@ -0,0 +1,24 @@
+---
+id: glossary
+title: Glossary
+sidebar_label: Glossary
+---
+
+<dl>
+  <dt>Activity Function</dt>
+  <dd></dd>
```

Comment:
This page seems incomplete. No definitions for any of the terms?

Reply:
Yea, I noted this above.

> I attempted to create a "Glossary" page but I think that's a WIP since there's a lot more going on that I'm still learning about. I've left it in for now but it will not appear in the sidebar yet.

Reply:
I also forgot to point out that I still plan on making more docs improvements, so the Glossary will come in a later iteration. I also plan on going through more pages. This PR just got so big that I didn't want to expand it even more.

Reply:
👍